src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1328,4 +1328,34 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
   <data name="InvokingCleanBlockNotSupported" xml:space="preserve">
     <value>Directly invoking the 'clean' block of a script block is not supported.</value>
   </data>
+  <data name="WDACParserConfigKeywordLogTitle" xml:space="preserve">
+    <value>Parser Configuration Keyword</value>
+  </data>
+  <data name="WDACParserConfigKeywordLogMessage" xml:space="preserve">
+    <value>The Configuration keyword would not allowed in ConstrainedLanguage mode for untrusted script.</value>
+  </data>
+  <data name="WDACParserClassKeywordLogTitle" xml:space="preserve">
+    <value>Parser Class Keyword</value>
+  </data>
+  <data name="WDACParserClassKeywordLogMessage" xml:space="preserve">
+    <value>The Class keyword would not allowed in ConstrainedLanguage mode for untrusted script.</value>
+  </data>
+  <data name="WDACParserDSSupportedCommandLogTitle" xml:space="preserve">
+    <value>Parser Data Section SupportedCommand</value>
+  </data>
+  <data name="WDACParserDSSupportedCommandLogMessage" xml:space="preserve">
+    <value>The Data Section that includes the SupportedCommand parameter would be disallowed in ConstrainedLanguage mode for untrusted script.</value>
+  </data>
+  <data name="WDACParserModuleScopeCallOperatorLogTitle" xml:space="preserve">
+    <value>Module Scope Call Operator</value>
+  </data>
+  <data name="WDACParserModuleScopeCallOperatorLogMessage" xml:space="preserve">
+    <value>The module scope call operator will be denied in policy enforcement.</value>
+  </data>
+  <data name="WDACParserForEachOperatorLogTitle" xml:space="preserve">
+    <value>ForEach Operator Method Invocation</value>
+  </data>
+  <data name="WDACParserForEachOperatorLogMessage" xml:space="preserve">
+    <value>The ForEach operator will fail '{0}' method invocation when run in Constrained Language mode.</value>
```

Comment:
There is not ForEach operator. It is either a keyword, a method name, or an alias for the cmdlet.
```suggestion
    <value>The ForEach() method will fail '{0}' method invocation when run in Constrained Language mode.</value>
```

Reply:
This applies to the keyword.